docs/en/cloud/reference/changelog.md

```diff
@@ -517,7 +517,7 @@ This release brings an officially supported Metabase integration, a major Java c
 ### Integrations changes
 - [Metabase](/docs/en/integrations/data-visualization/metabase-and-clickhouse.md) plugin: Became an official solution maintained by ClickHouse
 - [dbt](/docs/en/integrations/data-ingestion/etl-tools/dbt/index.md) plugin: Added support for [multiple threads](https://github.com/ClickHouse/dbt-clickhouse/blob/main/CHANGELOG.md)
-- [Grafana](/docs/en/integrations/data-visualization/grafana-and-clickhouse.md) plugin: Better handling of connection errors
+- [Grafana](/docs/en/integrations/data-visualization/grafana/index.md) plugin: Better handling of connection errors
```

Comment:
If you change a page URL, please update the redirect rules https://github.com/ClickHouse/clickhouse-docs/blob/47325ca4f4f9217f9bfbe16d1326f157927adcdb/docusaurus.config.js#L268